features/keyboard-lock.yml

```
@@ -0,0 +1,8 @@
+name: Keyboard lock
+description: Keyboard lock allows websites to capture keys that are normally reserved by the underlying host operating system. It is intended to be used by web applications that provide a fullscreen immersive experience.
```

https://github.com/web-platform-dx/web-features/tree/main/docs#allows discourages use of "allows" in most cases except "allows you". https://github.com/web-platform-dx/web-features/tree/main/docs#descriptions also recommends "The Thing TypeOfThing" for most descriptions. Applying both here:

```suggestion
description: The `navigator.keyboard.lock()` API allows you to capture keys that are normally reserved by the operating system. It can be used to provide an immersive fullscreen experience such as games.
```